src/base/bittorrent/session.cpp

```diff
@@ -4546,6 +4592,9 @@ void Session::readAlerts()
     const std::vector<lt::alert *> alerts = getPendingAlerts();
     for (const lt::alert *a : alerts)
         handleAlert(a);
+
+    if (!m_updatedTrackerEntries.isEmpty())
```

Comment:
maybe this check is redundant? just call the following function and let it do the work.